SpatialGDK/Source/SpatialGDKServices/Private/SSpatialOutputLog.cpp

```diff
@@ -272,39 +268,23 @@ void SSpatialOutputLog::StartPollTimer(const FString& LogFilePath)
 
 void SSpatialOutputLog::FormatAndPrintRawErrorLine(const FString& LogLine)
 {
-	const FRegexPattern ErrorPattern = FRegexPattern(TEXT("level=(.*) msg=(.*) code=(.*) code_string=(.*) error=(.*) stack=(.*)"));
-	FRegexMatcher ErrorMatcher(ErrorPattern, LogLine);
+	FString LogMessage = *LogLine;
```

Comment:
In this class you never actually modify the `LogLine` so why not just pass that into `Serialize` rather than making a copy of it here?